### ISSUE TITLE
[BUG] Graphical Glitch (COSMIC Terminal)

### ISSUE THREAD

**Issue by Querzion** [aristocratos/btop#890]
**Read the README.md and search for similar issues before posting a bug report!**

Any bug that can be solved by just reading the [prerequisites](https://github.com/aristocratos/btop#prerequisites) section of the README will likely be ignored.

**Describe the bug**

Graphical glitch. This could be a Distro problem too, so I will create an issue remark there too.

**To Reproduce**

Started btop in COSMIC Terminal in the COSMIC DE.

**Expected behavior**

Not with a graphical glitch.

**Screenshots**

![screenshot-2024-06-30-15-15-34](https://github.com/aristocratos/btop/assets/39269026/e6f4a5ce-bcb6-4d33-80ac-3ed5897e8c81)

**Info (please complete the following information):**
btop version: 1.2.3

neofetch output;

OS: Pop!_OS 22.04 LTS x86_64
Host: KVM/QEMU (Standard PC (Q35 + ICH9, 2009) pc-q35-7.2)
Kernel: 6.9.3-76060903-generic
Uptime: 50 mins
Packages: 1884 (dpkg), 6 (flatpak)
Shell: bash 5.1.16
Resolution: 2560x1440
DE: COSMIC
Theme: Pop-dark [GTK3]
Icons: Adwaita [GTK3]
Terminal: cosmic-term
CPU: AMD Ryzen 9 7900X (16) @ 4.699GHz
GPU: AMD ATI Radeon RX 6700/6700 XT / 6800M
Memory: 3493MiB / 32088MiB

**Additional context**

Contents of `~/.config/btop/btop.log`
2024/06/30 (17:27:14) | ===> btop++ v.1.2.3
2024/06/30 (17:27:14) | WARNING: No good candidate for cpu sensor found, using random from all found sensors.

**Comment by RedBearAK** [comment]
I had this issue on `cosmic-term` and checked out their GitHub issue. After installing the `Iosevka` font as suggested, apparently it is acting as a fallback font for the braille characters, but now with proper alignment. So, problem solved. 

You don't have to select the font as your terminal font, you just have to have it installed and available to provide the braille characters. Possibly the font where the braille characters were coming from before installing `Iosevka` was not a monospace font. That could explain the misalignment.

**Comment by rihards-simanovics** [comment]
> I had this issue on `cosmic-term` and checked out their GitHub issue. After installing the `Iosevka` font as suggested, apparently it is acting as a fallback font for the braille characters, but now with proper alignment. So, problem solved.
> 
> You don't have to select the font as your terminal font, you just have to have it installed and available to provide the braille characters. Possibly the font where the braille characters were coming from before installing `Iosevka` was not a monospace font. That could explain the misalignment.

Yep, I know it's been said, but I want to confirm that even in 2026 this is indeed the case, download iosevka-term variant and all looks good.
